flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -275,7 +275,14 @@
 	 * Path to Hadoop configuration
 	 */
 	public static final String PATH_HADOOP_CONFIG = "fs.hdfs.hadoopconf";
-	
+
+	// ------------------------ Storm Configuration ------------------------
+
+	/**
+	 * storm configuration
+	 */
+	public static final String STORM_DEFAULT_CONFIG = "storm.config";
```

Comment:
I think we should put this somewhere else... Maybe adding a new class in `stormcompatibility.util` package. @StephanEwen what is your opinion?